dataapi/clock.go

```diff
@@ -0,0 +1,46 @@
+package dataapi
+
+import (
+	"time"
+)
+
+// Clock - an implementation of a simple string DataItem that changes every second
+type Clock struct {
```

Comment:
This is a nice demo, but I think it would be better included in the demo code than in the actual API - the formatting and locale issues get complex fast ;)